app/services/geocoder/update_all_schools.rb

```diff
@@ -0,0 +1,18 @@
+module Geocoder
+  class UpdateAllSchools
+    include ServicePattern
+
+    def call
+      Rails.logger.info "Updating schools with longitude and latitude!"
+
+      School.not_geocoded.find_each do |school|
+        school.geocode
+        school.save!
+      rescue ActiveRecord::RecordInvalid => e
```

Comment:
Could we raise a specific error here, possibly a custom one so that we know it's to do with the Geocoding and not just a general invalid record?

Reply:
Resolved in [c33d97b](https://github.com/DFE-Digital/itt-mentor-services/pull/382/commits/c33d97b1f77a5bbdd5a6212b8e5e6bff47a6dd8d)